lib/recovery/recovery.h

```diff
@@ -19,8 +20,8 @@ namespace Recovery {
     drogue_chute_state = 0;
     main_chute_state = 0;
 
-    digitalWrite(drogue_pin, drogue_chute_state);
-    digitalWrite(main_pin, main_chute_state);
+    digitalWrite(DROGUE_PIN, drogue_chute_state);
+    digitalWrite(MAIN_CHUTE_PIN, main_chute_state);
```

Comment:
nope. update the init to take in two pin numbers, set the value of `drogue_pin` and `main_pin` to the values that were passed in . Use those two vars locally